src/test/java/core/basesyntax/service/RegistrationServiceTest.java

```diff
@@ -0,0 +1,72 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.RegistrationException; // Правильний імпорт винятку
+import core.basesyntax.model.User;
+import org.junit.Before;
+import org.junit.Test;
+
+public class RegistrationServiceTest {
+    private RegistrationService registrationService;
+    private StorageDaoImpl storage;
+
+    @Before
+    public void setUp() {
+        storage = new StorageDaoImpl(); // Використовуємо справжню реалізацію
+        registrationService = new RegistrationServiceImpl(storage);
+    }
+
+    @Test
+    public void register_validUser_ok() {
+        User validUser = new User("validLogin", "validPassword", 20);
+
+        registrationService.register(validUser);
+
+        assertEquals(validUser, storage.get(validUser.getLogin()));
```

Comment:
It's good that you've written a test for a valid user registration. However, according to the checklist, you should use low-level methods like `Storage.people.add()` directly instead of `RegistrationService.register()` method to add users for testing. This test should be modified to verify that after successful registration, the user was added to storage by checking the storage directly.